index.js

```diff
@@ -107,40 +108,30 @@ class KeyringController extends EventEmitter {
    *
    * @fires KeyringController#unlock
    * @param {string} password - The password to encrypt the vault with.
-   * @param {string|Array<number>} seedPhrase - The BIP39-compliant seed phrase,
-   * either as a string or an array of UTF-8 bytes that represent the string.
+   * @param {Uint8Array | string} seedPhrase - The BIP39-compliant seed phrase,
+   * either as a string or Uint8Array.
    * @returns {Promise<object>} A Promise that resolves to the state.
    */
   async createNewVaultAndRestore(password, seedPhrase) {
-    const seedPhraseAsBuffer =
-      typeof seedPhrase === 'string'
-        ? Buffer.from(seedPhrase, 'utf8')
-        : Buffer.from(seedPhrase);
+    const seedPhraseUint8Array = this._mnemonicToUint8Array(seedPhrase);
 
     if (typeof password !== 'string') {
       throw new Error('Password must be text.');
     }
 
-    const wordlists = Object.values(bip39.wordlists);
-    if (
-      wordlists.every(
-        (wordlist) => !bip39.validateMnemonic(seedPhraseAsBuffer, wordlist),
-      )
-    ) {
-      throw new Error('Seed phrase is invalid.');
+    if (!bip39.validateMnemonic(seedPhraseUint8Array, wordlist)) {
```

Comment:
we only validate against the english wordlist now.

Reply:
Makes sense.

For the changelog, is this a breaking change? i.e. did `eth-hd-keyring` ever support non-english mnemonics?

Reply:
No it did not. Back when we were using `bitcoinjs/bip39` and our fork [we would validate against](https://github.com/bitcoinjs/bip39/blob/f88d0dd4db96b3ccbba1c8bd6ec87bff54025949/src/index.js#L67) the `default` wordlist which was [set to english by default].(https://github.com/bitcoinjs/bip39/blob/f88d0dd4db96b3ccbba1c8bd6ec87bff54025949/src/_wordlists.js#L57).

Reply:
Great, thanks!